File: java/ConvertStringToArray.java
// convert a string into array of words

import java.util.Arrays;

public class ConvertStringToArray {

  public static String[] stringToArray(String s) {
    return s.split(" ");
  }

  public static void main(String[] args) {
    System.out.println(Arrays.toString(stringToArray("I love arrays they are my favorite"))); //
  }
}

/*
 * Write a function to split a string and convert it into an array of words.
 */

File: c#/ConvertStringToArray.cs
// convert a string to an array

using System;

public class ConvertStringToArray
{
	public static string[] StringToArray(string str) => str.Split(" ");

	public static void Main(string[] args)
	{
		Console.WriteLine(string.Join(" ", StringToArray("This is easy"))); // ["This", "is", "easy"]
	}
}